app.js

```diff
@@ -0,0 +1,105 @@
+// const books = [
+//     // {
+//     //     title: "The Valid Score",
+//     //     author: "Siri Amber",
+//     // },
+//     // {
+//     //     title: "Think and Grow Rich",
+//     //     author: "Napoleon Hill",
+//     // },
+//     // {
+//     //     title: "Rich Dad, Poor Dad",
+//     //     author: "Robert Kiyosaki",
+//     // },
+//     // {
+//     //     title: "Atomic Habits",
+//     //     author: "James Clear",
+//     // },
+//     // {
+//     //     title: "No Sweetness Here",
+//     //     author: "Ama Atta-Aiddo",
+//     // },
+// ]
+
+// get event listeners
+const titleInput = document.getElementById('title');
+const authorInput = document.getElementById('author');
+const form = document.querySelector('form');
+const booksContainer = document.querySelector('.books-container');
+
+// books collection
+let booksCollection = JSON.parse(localStorage.getItem('books-collection')) || [];
+
+function createCollection(book) {
+  const content = `
+        <div class="booklist">
+            <p>${book.title}</p>
+            <p>${book.author}</p>
+            <button class="removeBtn" id="${book.id}" type="submit">Remove</button>
+            <hr>
+        </div>
+    `;
+
+  return content;
+}
+
+function injectContent(arr) {
+  let books = '';
+
+  arr.forEach((object) => {
+    books += createCollection(object);
+  });
+  // console.log(books)
+  booksContainer.innerHTML = books;
+  // console.log(booksContainer.innerHTML)
```

Comment:
- [ ] Kindly remove all commented out, unused. unwanted pieces of code from your code base. It makes it look messy and cluttered.